spec/features/managing_employees_spec.rb

```diff
@@ -65,10 +71,46 @@
           fill_in :employee_street_address, with: address
           fill_in :employee_postal_code, with: postcode
           fill_in :employee_date_of_birth, with: dob
-          fill_in :employee_employed_from, with: employed_from
           page.select job_title, from: :employee_job_title
+          [
+            "Designated Behaviour Management Lead",
+            "Designated Safeguarding Lead",
+            "Designated SENCO",
+            "First Aider",
+            "Ofsted Registered Contact",
+            "Practice Leader"    
+          ].each do |r| 
+            if r ==  "Designated Safeguarding Lead" || r == "First Aider"
```

Comment:
Do we need this loop if we are just checking two roles?

Reply:
Yes, getting rid of that `if` definitely makes it less complicated to read! You could even simplify it further, as in the expectations you're only checking one of them ("Designated Safeguarding Lead") so you could get rid of the `map` and just check the one field.

Or if you wanted to test that saving two roles works, you could just have two lines with `check`s and that would also be easier to read than the `map`.

Reply:
Ahh ok! I made the change, and add few checks more